core/os/safe_binary_mutex.h

```diff
@@ -93,16 +93,39 @@ template <int Tag>
 class MutexLock<SafeBinaryMutex<Tag>> {
 	friend class ConditionVariable;
 
-	THREADING_NAMESPACE::unique_lock<THREADING_NAMESPACE::mutex> lock;
+	mutable THREADING_NAMESPACE::unique_lock<THREADING_NAMESPACE::mutex> _lock;
```

Comment:
Despite a few Godot classes are using leading underscore for private "fields," I think the most established rule is to use that only for function names.

Reply:
Was only named such because of the name clash with `lock()`, now it's unnecessary so will rename it